zrml/rikiddo/src/lib.rs

```diff
@@ -53,14 +53,14 @@ mod pallet {
 
     impl<T: Config> Pallet<T> {}
 
-    // This is the storage containing the LsdLmsr instances per pool.
+    // This is the storage containing the Rikiddo instances per pool.
     /*
     #[pallet::storage]
     pub type LmsrPerPool<T: Config> = StorageMap<
         _,
         Twox64Concat,
         u128,
-        LsdLmsrSigmoidMV<
+        RikiddoMV<
             FixedU128<T::FractionalType>,
             FeeSigmoid,
             EmaMarketVolume<FixedU128<T::FractionalType>>,
```

Comment:
More commented code. Guess they will be used in the future

Reply:
This is correct, will be used in one PR after the next one.